db/migrate/20231010201451_create_user_mutes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUserMutes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user_mutes do |t|
+      t.references :creator, :null => false
```

Comment:
I think we can add `:index => false` here as we have a separate index on creator+appointee for uniqueness so also having one on creator is unnecessary.